test/suite/test_tiered02.py

```diff
@@ -35,10 +35,10 @@
 #    Test tiered tree
 class test_tiered02(wttest.WiredTigerTestCase):
     storage_sources = [
-        ('local', dict(auth_token = get_auth_token('local_store'),
-            bucket = get_bucket1_name('local_store'),
+        ('local', dict(auth_token = get_auth_token('dir_store'),
```

Comment:
I don't have strong feelings but should the scenario now be called 'dir' instead of 'local'? The other is 's3' so it formerly was the extension name. (This would apply to every test, not just this one.)

Reply:
Yes, I missed these, and will fix.  Thanks!